koku/sources/api/urls.py

```diff
@@ -16,11 +16,12 @@
 """Describes the urls and patterns for the API application."""
 from django.conf.urls import url
 from sources.api.status import get_status
-from sources.api.views import billing_source
+from sources.api.views import authentications, billing_source
```

Comment:
Is there some reason authentication is plural while billing_source is singular?

Reply:
No reason.  Other then I had `/authentications` on the mind from what the Platform-Sources calls it.  I re-named this to `/authentication`